components/eamxx/src/physics/shoc/tests/shoc_eddy_diffusivities_tests.cpp

```diff
@@ -24,6 +24,8 @@ namespace unit_test {
 template <typename D>
 struct UnitWrap::UnitTest<D>::TestShocEddyDiff {
 
+  // TODO: Property tests need to be fixed to account for change of inputs (no obklen, add tabs)
+  /*
```

Comment:
Property tests for `eddy_diffusivities` and `shoc_tke` need to be updated for new inputs.